projects/wvr-elements/src/lib/wvr-dropdown/wvr-dropdown.component.ts

```diff
@@ -122,7 +380,7 @@ export class WvrDropdownComponent extends WvrAbstractBaseComponent {
    * And the click occured off of the wvr-dropdown component.
    */
   @HostListener('document:click', ['$event']) clickout($event): void {
-    if (!this.eRef.nativeElement.contains(event.target)) {
+    if (!this.eRef.nativeElement.contains($event.target)) {
```

Comment:
Remove this, as it relates to another issue

Reply:
$event used instead of event.